udm/intermediate/shopify-samples/orders-json/Backorder Expedited - content.json

```diff
@@ -0,0 +1,391 @@
+{
+  "order": {
+    "id": 5524202553597,
+    "admin_graphql_api_id": "gid://shopify/Order/5524202553597",
+    "app_id": 580111,
+    "browser_ip": "111.118.250.178",
+    "buyer_accepts_marketing": false,
+    "cancel_reason": null,
+    "cancelled_at": null,
+    "cart_token": "c1-ea3d4500278708b3ddaae3676785e6d9",
+    "checkout_id": 34877552820477,
+    "checkout_token": "8b296d2a5d75fd93c761fa52c23b689c",
+    "client_details": {
+      "accept_language": "en-IN",
+      "browser_height": null,
+      "browser_ip": "111.118.250.178",
+      "browser_width": null,
+      "session_hash": null,
+      "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"
+    },
+    "closed_at": null,
+    "company": null,
+    "confirmed": true,
+    "contact_email": "muskan@hotwax.com",
+    "created_at": "2024-02-12T06:02:43-05:00",
+    "currency": "USD",
+    "current_subtotal_price": "69.00",
+    "current_subtotal_price_set": {
+      "shop_money": {
+        "amount": "69.00",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "69.00",
+        "currency_code": "USD"
+      }
+    },
+    "current_total_additional_fees_set": null,
+    "current_total_discounts": "0.00",
+    "current_total_discounts_set": {
+      "shop_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      }
+    },
+    "current_total_duties_set": null,
+    "current_total_price": "75.90",
+    "current_total_price_set": {
+      "shop_money": {
+        "amount": "75.90",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "75.90",
+        "currency_code": "USD"
+      }
+    },
+    "current_total_tax": "0.00",
+    "current_total_tax_set": {
+      "shop_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      },
+      "presentment_money": {
+        "amount": "0.00",
+        "currency_code": "USD"
+      }
+    },
+    "customer_locale": "en-IN",
+    "device_id": null,
+    "discount_codes": [],
+    "email": "muskan@hotwax.com",
+    "estimated_taxes": false,
+    "financial_status": "authorized",
+    "fulfillment_status": null,
+    "landing_site": "/",
+    "landing_site_ref": null,
+    "location_id": null,
+    "merchant_of_record_app_id": null,
+    "name": "#2008",
+    "note": null,
+    "note_attributes": [],
+    "number": 1008,
+    "order_number": 2008,
+    "order_status_url": "https://hc-demo.myshopify.com/6973849727/orders/3a5e6651c2277ffe502c2f3202c75fb6/authenticate?key=df6f68fabac4c2f97073094e57a81c41",
```

Comment:
Do not commit the key, I;d suggest to use hc-sandbox url, please remove the key from url

Reply:
@sanjanashrivastava Replaced the value here